EasyLearn/app/controllers/users_controller.rb

```diff
@@ -19,8 +19,11 @@ def create
     end
   end
 
+  def myContent
```

Comment:
Remembre that in Ruby we use `lower_snake_case` to define variables, methods, etc.